### ISSUE TITLE
Window position/Always On Top do NOT reset upon app close

### ISSUE THREAD

**Issue by ryehamstrawberry** [AlexanderPro/SmartSystemMenu#145]
First, thank you for providing this utility.

I've never encountered an issue until recently when running an instance of KeePassXC, whose window position is not being saved on exit and which opens with SSM's Always On Top always enabled.

I've tried manually resizing the window and moving it to the desired position, disabling Always On Top, then exiting KeePassXC via the menu.
For whatever reason, KeePassXC will continue to open itself in `position left="1274" top="31" width="1294" height="1417"` and `alwaysOnTop="true"` despite Window64.xml being empty.

The only solution I've found is to set my window position, disable Always On Top, then manually select "Save" from the context menu.
While this achieves the result I want, this is an inefficient workaround because I believe this should be possible without having to save per-app settings for this one app.
None of my other apps are affected in the same way, they all behave correctly by opening in the app's last used position and without Always On Top being enabled by default.

I have a feeling I'm overlooking something but I'm unsure where else to look.
Has anyone else experienced SSM per-app settings being erroneously saved and is there a solution?

**Comment by AlexanderPro** [comment]
Hi @ryehamstrawberry,

I have downloaded KeePassXC v2.7.9 and I've tried to reproduce the issue but I have not been able to. I have checked "Always On Top" menu item, then unchecked it and after restart the process its window is not "Always On Top". Сould you please describe the process step by step or attach a video or gif file?

**Comment by ryehamstrawberry** [comment]
Hey @AlexanderPro,
I think I resolved the issue by disabling the native KeePassXC "Always On Top" (Ctrl+Shift+A) option, which I didn't realize existed.

Upon experimentation, it seems that KeePassXC's AOT setting take precedent over SSM's AOT setting, as it should.

tl;dr: user error is responsible, I believe this issue is closed.
